types/commerce7__admin-ui/commerce7__admin-ui-tests.ts

```diff
@@ -0,0 +1 @@
+import { Commerce7AdminUI } from "@commerce7/admin-ui";
```

Comment:
This seems like not very many tests; should this be a `.tsx` file and then actually use the components?

Reply:
Yeah, I will admit I didn't really know what tests to write. Do you know of a react component library that is similar that I could use for some guidance? I looked through the typings for material-ui, but that library seems so different than @commerce7/admin-ui that the tests didn't look relevant.

Reply:
I don't have any specific examples, but if I were looking I would just be finding another package with a `.tsx` file.

Reply:
Good idea, I will try that once I have some more time to work on this. Thanks for the nudge in the right direction.